src/Resources/Locales/en.json

```diff
@@ -1111,6 +1111,30 @@
         "lastRefreshed": "Last refreshed: {{value}} {{units}} ago",
         "refreshRate": "Refresh rate: {{value}} {{units}}"
     },
+    "sceneRefreshConfigurator": {
+        "headerButtonTitle": "Scene configuration",
+        "calloutTitle": "Scene configuration",
+        "calloutDescription": "Customize how often this scene's data is refreshed",
+        "strategyDropdownLabel": "Refresh strategy",
+        "strategyTooltip": "Use 'Realtime' to fetch data as quickly as possible (max: every 10 seconds). \nUse 'Limited' to select a rate limit that limits the frequency of refreshes to a slower rate.",
```

Comment:
nit: maybe would be pithier to say something like 
"Use 'Realtime' to fetch data as quickly as possible (with a max. rate of 10 seconds). Use 'Limited' to limit the frequency of refreshes to a slower rate.

Reply:
yesssss. thank you. This is much better.